react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -111,7 +128,18 @@ class OverflowMenu extends Component<Props> {
                 }
                 <RaiseHandButton { ...buttonProps } />
                 <SharedDocumentButton { ...buttonProps } />
-                <HelpButton { ...buttonProps } />
+                <OpenUrlButton
+                    { ...buttonProps }
+                    _externalUrl = { this.props._userDocumentationUrl }
+                    accessibilityLabel = { t('toolbar.accessibilityLabel.help') }
+                    icon = { IconHelp }
+                    label = { t('toolbar.help') } />
+                <OpenUrlButton
```

Comment:
What sense does it make to have a button to download the apps from within the app itself?

Reply:
I guess it doesn't necessarily make much sense in the mobile context, will leave it out of the mobile overflow menu.